app/components/Home.tsx

```diff
@@ -0,0 +1,21 @@
+import AlignmentFeedTable from "~/components/AlignmentFeedTable";
+
+export default function Home({ apiBaseURL, darkMode }: { apiBaseURL: string, darkMode:string }) {
+    return (
+        <>
+            <div className='h-screen w-full flex flex-col space-y-4 pb-5 pt-3'>
```

Comment:
Because this is h-screen, but the navbar is also taking up vertical height, when the page is opened the viewport sizes itself to always the screen + navbar in height- might it be worth tweaking this so we fit screen (maybe with a minimum for really small displays)?

I think making this 100% and the parent of both it and the navbar h-screen when looking at this page might work.